### ISSUE TITLE
Maui support?

### ISSUE THREAD

**Issue by IeuanWalker** [SKLn-Rad/Xam.Plugin.Webview#164]
Just wondering if this is going to migrated to MAUI?

**Comment by jdinnen** [comment]
Yeah, I would like something like this in Maui too but this hasn't been updated in 4 years so would not hold my breath...
